web-stories.php

```diff
@@ -41,7 +41,7 @@
 	exit;
 }
 
-define( 'WEBSTORIES_VERSION', '1.0.0-alpha' );
+define( 'WEBSTORIES_VERSION', '1.0.0-alpha+773f1c4' );
```

Comment:
Hmm I need to fix the `npm run build` command

Reply:
Build seems to be broken. The font file is dealing itself as well.